packages/base/src/experiment-manager.ts

```diff
@@ -6,13 +6,14 @@ import { Experiment } from 'tsp-typescript-client/lib/models/experiment';
 import { TraceManager } from './trace-manager';
 import { TspClientResponse } from 'tsp-typescript-client/lib/protocol/tsp-client-response';
 import { signalManager, Signals } from './signals/signal-manager';
+import { TspFrontendClient } from './tsp-frontend-client';
 
 export class ExperimentManager {
     private fOpenExperiments: Map<string, Experiment> = new Map();
-    private fTspClient: TspClient;
+    private fTspClient: TspClient | TspFrontendClient;
```

Comment:
There are a lot of cases where this type option has been added. Would it make sense to define a single interface that both services satisfy and just refer to that?

Reply:
A converse to [a comment](https://github.com/eclipse-cdt-cloud/tsp-typescript-client/pull/83/files#r1270706958) I've just made over there: perhaps the `TspFrontendClient` could be named more precisely to something like `TheiaRpcClient` (still implementing the new `TspClient` interface)?